internal/service/reclaimspace/reclaimspace.proto

```diff
@@ -8,16 +8,16 @@ option go_package = "github.com/csi-addons/kubernetes-csi-addons/internal/servic
 service ReclaimSpace {
     // ControllerReclaimSpace is a procedure that gets called on the CSI
     // sidecar.
-    rpc ControllerReclaimSpace (ControllerReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+    rpc ControllerReclaimSpace (ReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
     
     // NodeReclaimSpace is a procedure that gets called on the CSI
     // sidecar.
-    rpc NodeReclaimSpace (NodeReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
+    rpc NodeReclaimSpace (ReclaimSpaceRequest) returns(ReclaimSpaceResponse) {}
```

Comment:
Changing package name and regenerating the files seems to work. 
@yati1998 can you modify package name instead please?

@nixpanic @yati1998  I am also inclined to keep a single `RecaimSpaceRequest` similar to `ReclaimSpaceResponse` since they share the same parameter.

`StorageUsage` can be reverted back to `StorageConsumption`

Reply:
There needs to be a way for the side-car to make a distinction between NodeReclaimSpace and ControllerReclaimSpace. Using different procedures makes that the cleanest.

Reply:
Yes, I referred to the `Request` struct, holding pv_name.

Reply:
yes, the rpc's names are different only are arguments are same. In my opinion, it will be fine, isn't it???